app/src/main/java/com/framgia/englishconversation/utils/binding/BindingUtils.java

```diff
@@ -26,10 +26,16 @@
 
 import com.bumptech.glide.Glide;
 import com.framgia.englishconversation.R;
+import com.framgia.englishconversation.data.model.UserModel;
+import com.framgia.englishconversation.data.source.callback.DataCallback;
```

Comment:
ERROR: Unused import: com.framgia.englishconversation.data.source.callback.DataCallback. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports